xchange-core/src/main/java/org/knowm/xchange/derivative/FuturesContract.java

```diff
@@ -97,8 +97,7 @@ public int hashCode() {
   @Override
   public String toString() {
 
-    return currencyPair
-        + "/"
+    return currencyPair.base + "-" + currencyPair.counter + "-" 
         + (expireDate == null ? PERPETUAL : DATE_PARSER.get().format(this.expireDate));
```

Comment:
again, think moving to hyphens will break things for a lot of people

Reply:
hyphens have returned